styleguide/components/31-molecules/filter-placeholder/filter-placeholder.twig

```diff
@@ -0,0 +1,5 @@
+<div class="filter-placeholder box-no-icon">
+  <p>{{ description }}</p>
+
+  <span class="spotimage"></span>
```

Comment:
@delrueba Is this okay for accessibility? This image is now invisible to screen readers... not sure if that's an issue here...?

Reply:
Sure, it's decorative. 
"Abstract blue lightbulb" doesn't add much information.

Reply:
Okay, thought so, but just checking :)